extensions/cornerstone-dicom-sr/src/getSopClassHandlerModule.ts

```diff
@@ -216,6 +216,12 @@ async function _load(
   });
 }
 
+function _measurementSharesFrameOfReferenceUID({ measurement, displaySet }) {
```

Comment:
Can you call it _measurementBelongsToDisplaySet
That way you can test for other things than the referenced frame of reference - in specific, if we restore the instance SOP UID, it becomes possible to restore it to the correct volume rather than just any volume with the same frame of reference uid.